ui-discover/src/main/java/app/tivi/home/discover/DiscoverTextCreator.kt

```diff
@@ -17,12 +17,15 @@
 package app.tivi.home.discover
 
 import android.content.Context
+import androidx.compose.runtime.staticAmbientOf
 import app.tivi.data.entities.Episode
 import app.tivi.data.entities.Season
 import dagger.hilt.android.qualifiers.ActivityContext
 import javax.inject.Inject
 
-internal class DiscoverTextCreator @Inject constructor(
+val DiscoverTextCreatorAmbient = staticAmbientOf<DiscoverTextCreator>()
```

Comment:
Ambient feels a bit much for this. Could just make an extension method on `Context`?